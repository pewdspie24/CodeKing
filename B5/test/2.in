6
3 10
7 4
8 5
4 5
7 2
2 3
