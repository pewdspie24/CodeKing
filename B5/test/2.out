512
279936
36
7
625
32
32768
